Test/TestsControllers/TestsApiControllers/QuoteControllerTests.cs

```diff
@@ -70,6 +70,10 @@ public void TestControllerMethodAttributes()
             authAttribute.ElementAt(0).Policy.ShouldBe(AccessCodes.SupervisorAccess);
             ControllerReflection.MethodExpectedAttribute<HttpPostAttribute>("Save", countAdjustment + 3);
             ControllerReflection.MethodExpectedAttribute<AsyncStateMachineAttribute>("Save", countAdjustment + 3);
+
+            //4
+            ControllerReflection.MethodExpectedAttribute<HttpGetAttribute>("GetApproved", countAdjustment + 2);
+            ControllerReflection.MethodExpectedAttribute<AsyncStateMachineAttribute>("GetApproved", countAdjustment + 2);

```

Comment:
Need to add a test for the details too.

Reply:
I'm just going off the established pattern. Where are the detail tests? I just see a bunch of ControllerReflection boilerplate.

Reply:
Details is just a get, with no async, so would look like this:
`//5
            ControllerReflection.MethodExpectedAttribute<HttpGetAttribute>("Details", 1);`

Reply:
🦤 I completely forgot about the Details method. Thought you were asking for a functional test.